pytest_html/plugin.py

```diff
@@ -288,7 +290,9 @@ def generate_summary_item(self):
         summary = [html.h2('Summary'), html.p(
             '{0} tests ran in {1:.2f} seconds. '.format(
                 numtests, suite_time_delta)),
-            html.p('(Un)check the boxes to filter the results.')]
+            html.p('(Un)check the boxes to filter the results.',
+                   id='filter_checkbox_text',
```

Comment:
We can change the checkboxes to use `class_='filter'` or similar, and reuse the class here. Then in the JavaScript we can just find elements with that class name and show them all. There would be no need to add the `id` here then.